index.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="style.css">
+    <link rel="shortcut icon" href="./images/google.png" type="image/x-icon">
+    <title>Google</title>
+</head>
+<body>
+
+    <div class="userinfo">
+        <ul>
+            <li> <a href="#">Gmail</a></li>
+            <li> <a href="#">Images</a></li>
+            <li> 
+                <div class="tooltip">
+                    <img src="./images/appGoogle.png" alt="google app" class="googleapp"><span class="toolTipText"> Google apps</span>
+                </div>
+                </li>
+            <li> 
+                <div class="tooltipletter">
+                    <span class="letterborder">K</span> <span class="tooltipemail"><span class="ga"> Google Account</span> <br/> Kanu Mike Chibundu <br/> Michotall95@gmail.com</span>
+                </div>
+            </li>
+        </ul>
+    </div>
```

Comment:
- [ ] I think it would be better to use semantic tags by using `<nav>` instead of `<div>`, that would improve the HTML structure and make the page more accessible